core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -305,6 +305,23 @@ public T maxInboundMessageSize(int bytes) {
     return thisT();
   }
 
+  /**
+   * Sets the maximum size of metadata allowed to be received. {@code Integer.MAX_VALUE} disables
+   * the enforcement. The default is implementation-dependent, but is not generally less than 8 KiB
+   * and may be unlimited.
+   *
+   * <p>This is cumulative size of the headers. The precise calculation is implementation-dependent,
+   * but implementations are encouraged to follow the calculation used for
+   * <a href="http://httpwg.org/specs/rfc7540.html#rfc.section.6.5.2">
+   * HTTP/2's SETTINGS_MAX_HEADER_LIST_SIZE</a>. It sums the bytes from each entry's key and value,
+   * plus 32 bytes of overhead per entry.
+   */
+  public NettyChannelBuilder maxInboundMetadataSize(int maxInboundMetadataSize) {
```

Comment:
nit: was the parameter supposed to be named `bytes` for this set of functions? For InProcess it is currently named  `bytes`.